docs/gen_ref_pages.py

```diff
@@ -1,111 +1,58 @@
 """
 Generate the code reference pages.
```

Comment:
What about adding a comment that tell that this script is executed when running `mkdocs serve -f docs/mkdocs.yml`?